.github/workflows/phpunit.yml

```diff
@@ -122,7 +122,7 @@ jobs:
 
       - name: Set up PHPUnit
         working-directory: apps/${{ env.APP_NAME }}
-        run: composer i
+        run: composer install && composer bin phpunit install
```

Comment:
could you do that through the hook? https://github.com/nextcloud/mail/blob/dff7a2751ca340b9f104d9103d291d1bd83aff73/composer.json#L58